OpenSim/Actuators/Millard2012AccelerationMuscle.h

```diff
@@ -897,27 +900,34 @@ OpenSim_DECLARE_CONCRETE_OBJECT(Millard2012AccelerationMuscle, Muscle);
                                           const AccelerationMuscleInfo& ami,
                                           std::string& caller) const;
 
-    //=====================================================================
-    // Private Utility Class Members
-    //      -Computes activation dynamics and fiber kinematics
-    //=====================================================================
+    // Status flag returned by initMuscleState().
+    enum ResultOfInitMuscleState {
+        Success_Converged,
+        Warning_FiberAtLowerBound,
+        Failure_MaxIterationsReached
+    };
+
```

Comment:
If this enum can be generalized to be applicable to other all muscles, consider moving the enum out of the class and making it an `enum class` (It offers lot more type safety and regular enum too).
Moving it out of the class may have implications toward bindings.